android/src/main/kotlin/com/signify/hue/flutterreactiveble/ble/ReactiveBleClient.kt

```diff
@@ -227,10 +228,16 @@ open class ReactiveBleClient(private val context: Context) : BleClient {
                     deviceConnection.rxConnection.discoverServices()
                             .flatMap { deviceServices -> deviceServices.getCharacteristic(characteristic) }
                             .flatMapObservable { char ->
+                                val mode = if (char.descriptors.isEmpty()) {
+                                    NotificationSetupMode.COMPAT
+                                } else {
+                                    NotificationSetupMode.DEFAULT
+                                }
+
```

Comment:
I saw this part in another pull-request. What's going on here...

Reply:
Ah, I see. Different target branches.

Reply:
Since we now need to maintain two branches due to null-safety I decided to migrate it twice

Reply:
Yeah, but this will complicate rebasing. Better try rebasing it.